sample/src/main/res/values/colors.xml

```diff
@@ -14,6 +14,7 @@
      limitations under the License.
```

Comment:
Add Apache copyright in top as in other xml files

Reply:
Added